src/averageAges.js

```diff
@@ -14,14 +14,20 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+// - Here is some crazy solutions
+// |-- to solve it with no loops, if\else statements etc
+// |-- ...but a lot of variables))
+function calculateMenAverageAge(people, century = true) {
+  const men = people.filter(person => person.sex === 'm');
+  const ages = men
+    .map((person) =>
+      Math.ceil(person.died / 100) === century || century === true
```

Comment:
century is a number, I believe you don't need `=== true` for checking century
```suggestion
      Math.ceil(person.died / 100) === century || century
```

Reply:
@Denchepornyuk I fixed everything like logic, separate function, less methods and vars, BUT it doesnt work with out this `century === true` check))